frontend/src/pages/projects/screens/spawner/storage/SpawnerMountPathField.tsx

```diff
@@ -0,0 +1,153 @@
+    onChange(`${prefix}${pathSuffix}`);
+  };
+
+  const handleSuffixChange = (newSuffix: string) => {
+    const prefix = format === MountPathFormat.STANDARD ? MOUNT_PATH_PREFIX : '/';
+    onChange(`${prefix}${newSuffix}`);
+    setError(validateMountPath(newSuffix, inUseMountPaths || []));
+  };
+
+  return (
+    <FormGroup
+      label="Mount path"
+      fieldId="mount-path"
+      isRequired
+      labelIcon={
+        <FieldGroupHelpLabelIcon
+          content={
+            <>
+              The directory within a container where a volume is mounted and accessible. Only
+              standard paths that begin with <b>{MOUNT_PATH_PREFIX}</b> are visible in the
+              JupyterLab file browser.
+            </>
+          }
+        />
+      }
+    >
+      <Stack hasGutter>
+        {isCreate ? (
+          <>
+            <StackItem>
+              <Flex>
+                <FlexItem>
+                  <Radio
+                    id="mount-path-radio-standard"
+                    label="Standard path"
+                    name="mount-path-format-radio"
+                    value={MountPathFormat.STANDARD}
+                    isChecked={format === MountPathFormat.STANDARD}
+                    onChange={() => handleFormatChange(MountPathFormat.STANDARD)}
+                  />
+                </FlexItem>
+                <FlexItem>
+                  <Radio
+                    id="mount-path-radio-custom"
+                    label="Custom path"
+                    name="mount-path-format-radio"
+                    value={MountPathFormat.CUSTOM}
+                    isChecked={format === MountPathFormat.CUSTOM}
+                    onChange={() => handleFormatChange(MountPathFormat.CUSTOM)}
+                  />
+                </FlexItem>
+              </Flex>
+            </StackItem>
+            <StackItem>
+              <InputGroup>
+                <InputGroupText id="path-prefix" aria-label="Mount path prefix">
+                  {format === MountPathFormat.STANDARD ? MOUNT_PATH_PREFIX : '/'}
+                </InputGroupText>
+                <InputGroupItem isFill>
+                  <TextInput
+                    id="mount-path-input"
+                    data-testid="mount-path-folder-value"
+                    aria-label="Mount path suffix"
+                    type="text"
+                    value={pathSuffix}
+                    onChange={(_, value) => handleSuffixChange(value)}
+                    isRequired
+                    validated={error ? 'error' : 'default'}
+                  />
+                </InputGroupItem>
+              </InputGroup>
+              <HelperText>
+                <HelperTextItem
+                  variant={error ? 'error' : 'default'}
+                  data-testid="mount-path-folder-helper-text"
+                >
+                  {error || 'Must consist of lowercase letters and dashes.'}
```

Comment:
It would be better to say:

```
Must only consist of lowercase letters, dashes, and slashes.
```

`Must consist of lowercase letters and dashes.` implies that it MUST contain BOTH lower case letters and dashes.  If a user doesn't want to use a dash, that is perfectly acceptable.

As a comparison, if you were describing a password policy and you require the password to contain a special character you might say `Must contain lowercase letters and a special character`.  However, if you were describing what characters are allowed without requiring specific things you might say `Must only contain lowercase letters and special characters`.

Dashes are obviously not required to be present so it makes more sense to include `Must only`.